napari/layers/utils/plane.py

```diff
@@ -35,10 +35,6 @@ def _normalise_vector(cls, v):
     def _ensure_tuple(cls, v):
         return tuple(v)
 
-    def shift_along_normal_vector(self, distance: float):
-        """Shift the plane along its normal vector by a given distance."""
-        self.position += distance * self.normal
-
```

Comment:
I guess that was public, I guess even if unused we might want a deprecation ... @napari/core-devs ?

Reply:
This class is also exposed through the `Image.plane` property (and maybe others), but I also see it's generally broken right now. I guess if you pass through a positive integer distance, the code will not fail, but it also won't do what's expected. Maybe we could just fix it by iterating over the elements and multiplying as I assume is intended.

@brisvag : what are your thoughts about this?

Reply:
If this is used at all, it's probably by our plane example or maybe by @alisterburt in napari-threedee. But why does it need removal? I think this was left as a utility for plugins and users.

Reply:
Thanks, I have no memory of why I removed it... merged your PR to my fork, so should be good now.